ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml

```
@@ -245,6 +245,79 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build04
+  cron: 00 02 */1 * 1-4
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: assisted-service
+  labels:
+    ci-operator.openshift.io/cloud: packet-edge
+    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/variant: edge
+    ci.openshift.io/generator: prowgen
+    job-release: "4.12"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-hypershift-zero-nodes-periodic
+  spec:
```

sorry about it. for now we have to add this manually:
```suggestion
  reporter_config:
    slack:
      channel: '#assisted-deployment-ci'
      job_states_to_report:
      - failure
      - error
      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs>'
  spec:
```

---

hopefully soon we'll be able to have a global configuration of this ``reporter_config`` and we won't need to add more (https://github.com/openshift/ci-tools/pull/3140)

---

> hopefully soon we'll be able to have a global configuration of this `reporter_config` and we won't need to manually edit this part ([openshift/ci-tools#3140](https://github.com/openshift/ci-tools/pull/3140))

yep, would indeed be nicer:)